micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -43,6 +45,9 @@
 @NonNullApi
 @NonNullFields
 public class ExecutorServiceMetrics implements MeterBinder {
+
+    private final static InternalLogger log = InternalLoggerFactory.getInstance(ExecutorServiceMetrics.class);
```

Comment:
Oops, I forgot this: `static final` instead of `final static`:
```suggestion
    private static final InternalLogger log = InternalLoggerFactory.getInstance(ExecutorServiceMetrics.class);
```

Reply:
@jonatan-ivanov Thanks again. I applied it in 9c7f6e4.

For the record, it was a copy-and-paste, so there are more. I'll polish them separately.